testsuite/MDAnalysisTests/parallelism/test_multiprocessing.py

```diff
@@ -134,15 +134,18 @@ def create_universe():
 def test_creating_multiple_universe_without_offset():
     #  test if they can be created without generating
     #  the offset simultaneously.
+    #  The tested XTC file is way too short to induce a race scenario
     try:
         os.remove(XDR.offsets_filename(XTC))
```

Comment:
This remove interferes with other tests that may work with the XTC trajectory. You'd need to make a temporary copy first for this test.

Reply:
fixed in [94611d7](https://github.com/MDAnalysis/mdanalysis/pull/3375/commits/94611d79eab7d7e289410ce2649e184581a8f471)